SETUP/dump_db_schema.template

```diff
@@ -18,6 +18,10 @@ export LC_ALL=POSIX
 # So how will we determine which ones to dump?
 if [ "$1" = '--table-list-from-db' ]; then
     table_list_source='db'
+    # If they've passed in another arg, assume it's the DB name
```

Comment:
This might be an opportunity to put `set -euo pipefail` at the top of this template, but to do that we're going to need to change things that rely on expanding undefined variables. The `set -u` safe way of spelling `if [ ! -z "$foo" ]` is `if [ -n "${foo-}" ]`.
 `if [ -n "${2-}" ]; then` here, and `elif [ "${1-}" = "--table-list-from-code" ]; then` and similar above and below.

Reply:
Also perhaps make this whole if...elif..else chain into
```
case ${1-} in
  --table-list-from-db)
   ...;;
  --table-list-from-code)
   ...;;
   *)
   ...;;
esac
```

Reply:
Got `set -euo pipefail` added to both scripts. Turns out the "get db list from code" was subtly broken so I fixed that too.